firestore.rules

```diff
@@ -0,0 +1,17 @@
+
+// Allow read/write access on all documents to any user signed in to the application
+//service cloud.firestore {
+//  match /databases/{database}/documents {
+//    match /{document=**} {
+//      allow read, write: if request.auth != null;
```

Comment:
wouldn't we want this in? we are the only ones who should be able to read from and write to db